pom.xml

```diff
@@ -246,6 +246,13 @@
                 <artifactId>postgresql</artifactId>
                 <version>9.1-901-1.jdbc4</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.opentable.components</groupId>
+                <artifactId>otj-pg-embedded</artifactId>
+                <version>0.4.4</version>
```

Comment:
This is in `pom.xml` `dependencyManagement` which doesn't need a scope -- the scope is defined in `pg/pom.xml`